Server/src/main/java/local/AssetDAOImpl.java

```diff
@@ -228,6 +228,7 @@ public Asset createAssetFromQueryResult(ResultSet assetsQuery) throws SQLExcepti
         newAsset.setSerialNo(assetsQuery.getString("sn"));
         newAsset.setRecommendation(assetsQuery.getString("recommendation"));
         newAsset.setAssetTypeName(assetsQuery.getString("asset_type.name"));
+        newAsset.setImageId(assetsQuery.getInt("imageId"));
```

Comment:
This is added in this creation of asset method, but was not added in the other asset creation method on line 244. I know some of it is outside of the scope of this issue, but it seems like the 2 methods are doing the exact same thing. I think it would be good to delete one of the 2 and replace it with the other one where it was used. This way, you also only have to add the setImageId line at one specific place

Reply:
Great catch, I did not see this one. I'll remove it in my next commit.